k8s/events-monitor/prod-events-notifs-cronjob.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: prod-events-notifs-job
+  namespace: staging
```

Comment:
Same namespace as the one for staging, intentional? @NoahNsimbe

Reply:
thanks for the catch, let me make the change